nomos-services/tracing/src/lib.rs

```diff
@@ -237,18 +236,12 @@ where
             });
         }
 
-        #[cfg(test)]
         ONCE_INIT.call_once(move || {
```

Comment:
While I understand having it this way, I don't understand why we had them under macros in the first place. Is there some difference?